beps/bep_0003.rst

```diff
@@ -87,70 +92,153 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layers``
```

Comment:
Handshaking is unrelated. Clients have to do 2 announces / dht lookups anyway since the hashes change, so they know which peer supports which format. The issue is the chain-of-trust from the hash to the hybrid data. If the new hash is only derived from the new data but not from the hybrid format then metadata sourced from the new hash can't be used to talk to old clients since the old data cannot be obtained due to a lack of trust.